cluster-scope/base/core/namespaces/seraph-platform-mq/resourcequota.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ResourceQuota
+apiVersion: v1
+metadata:
+  name: custom
```

Comment:
can you change the name to `seraph-platform-mq-custom` so it's consistent with the other custom quota naming